src/components/Receive/AddressView.js

```diff
@@ -1,338 +1,98 @@
 // @flow
 
-import React, {useEffect, useState, useCallback} from 'react'
-import {connect} from 'react-redux'
-import {compose} from 'redux'
-import {withStateHandlers, withHandlers} from 'recompose'
-import {View, TouchableOpacity, Image, Platform} from 'react-native'
-import {injectIntl, type IntlShape, defineMessages} from 'react-intl'
+import React, {useEffect, useState} from 'react'
+import {View, TouchableOpacity, Image, StyleSheet} from 'react-native'
 import Clipboard from '@react-native-community/clipboard'
 
-import {
-  isUsedAddressIndexSelector,
-  externalAddressIndexSelector,
-  hwDeviceInfoSelector,
-  walletMetaSelector,
-} from '../../selectors'
-import {showErrorDialog} from '../../actions'
-import {setLedgerDeviceId, setLedgerDeviceObj} from '../../actions/hwWallet'
-
-import {Text, Modal} from '../UiKit'
-import AddressModal from './AddressModal'
-import LedgerTransportSwitchModal from '../Ledger/LedgerTransportSwitchModal'
-import LedgerConnect from '../Ledger/LedgerConnect'
-import AddressVerifyModal from './AddressVerifyModal'
-import {verifyAddress} from '../../crypto/shelley/ledgerUtils'
-import walletManager from '../../crypto/walletManager'
-import {formatPath} from '../../crypto/commonUtils'
-import {errorMessages} from '../../i18n/global-messages'
-import LocalizableError from '../../i18n/LocalizableError'
-import {Logger} from '../../utils/logging'
-import {CONFIG} from '../../config/config'
-import {getCardanoByronConfig} from '../../config/networks'
-import {AddressDTOCardano} from '../../crypto/shelley/Address.dto'
-
-import styles from './styles/AddressView.style'
+import {Text} from '../UiKit'
 import copyIcon from '../../assets/img/icon/copy-ext.png'
-import verifyIcon from '../../assets/img/icon/verify-address.png'
 import copiedIcon from '../../assets/img/icon/copied.png'
+import verifyIcon from '../../assets/img/icon/verify-address.png'
 
-import type {ComponentType} from 'react'
-import type {HWDeviceInfo, DeviceId, DeviceObj} from '../../crypto/shelley/ledgerUtils'
-import type {WalletMeta} from '../../state'
-
-const messages = defineMessages({
-  copyLabel: {
-    id: 'components.receive.addressmodal.copyLabel',
-    defaultMessage: '!!!Copy',
+import {type ViewProps} from 'react-native/Libraries/Components/View/ViewPropTypes'
+
+const styles = StyleSheet.create({
```

Comment:
In most places we have the styles apart from the components, do you think we should move everything inside?